flink/flink-1.11/src/test/java/io/tidb/bigdata/flink/tidb/FlinkTest.java

```diff
@@ -253,7 +267,7 @@ public void testTableFactory() throws Exception {
     row1.setField(0, LocalDateTime.of(2020, 1, 1, 12, 0, 1));
     row1.setField(1, "2020-01-01 12:00:02");
     Assert.assertEquals(row, row1);
-
+    
```

Comment:
Can you remove these dummy spaces from changeset?
There are couple of these in other files as well.

Reply:
IDE auto-formatting problem, I modify it